addons/account/models/chart_template.py

```diff
@@ -933,6 +933,32 @@ def _get_tax_vals(self, company, tax_template_to_tax):
             val['tax_group_id'] = self.tax_group_id.id
         return val
 
+    def _get_tax_vals_complete(self, company):
+        """
+        Returns a dict of values to be used to create the tax corresponding to the template, assuming the
+        account.account objects were already created.
+        Function _get_tax_vals is used when the account.account are not yet created. Thus, we need to add the
+        missing field cash_basis_transition_account_id. The invoice and refund repartition lines are also updated
+        so that they contain the following account.account objects: account_id, invoice_tax_id and refund_tax_id.
```

Comment:
not invoice_tax_id and refund_tax_id anymore, only the account. 

I'd replace the whole accounts explanation by simply saying we replace references to account templates by the corresponding account.account ids.